Solutions/ThreatXCloud/Playbooks/ThreatXPlaybooks/ThreatX-Enrichment/azuredeploy.json

```diff
@@ -31,31 +32,24 @@
         "customer_name": {
```

Comment:
I would suggest not to use underscore like other parameters.

Reply:
Done